NuRadioMC/SignalProp/analyticraytracing.py

```diff
@@ -1515,7 +1515,7 @@ class ray_tracing:
                       3: 'reflected'}
 
     def __init__(self, x1, x2, medium, attenuation_model="SP1", log_level=logging.WARNING,
-                 n_frequencies_integration=6,
+                 n_frequencies_integration=100,
```

Comment:
Why do we have a new standard and why is it so much larger?

Reply:
the smaller number can lead to uncertainties. Better a precise result that might be a bit slower than a wrong result that is fast.